### ISSUE TITLE
dropout / label_smoothing

### ISSUE THREAD

**Issue by vince62s** [tloen/alpaca-lora#347]
Hi,
Just a quick question.
I understand the finetuning goes with LoRa dropout of 0.05 but
what is the status of dropout (since it's training) of all other layers and embedding of the main model ? it is by default 0.1 ? set to 0 ?
same for laebel smoothing (usually set to 0.1) does peft set it to 0 ?
thx

**Comment by lywinged** [comment]
There are no other layers.
laebel smoothing =0

**Comment by vince62s** [comment]
In the llama architecture (from repo) there is none because it is an inference decoder architecture.
But in the GPT decoder there are dropout at the embedding level and for fully connected layers before the ff.
since we forward with the main model, hence I would expect some kind of dropout during training at those levels.

**Comment by lywinged** [comment]
> In the llama architecture (from repo) there is none because it is an inference decoder architecture.
> But in the GPT decoder there are dropout at the embedding level and for fully connected layers before the ff.
> since we forward with the main model, hence I would expect some kind of dropout during training at those levels.

LORA makes the pre-trained model's weight frozen, and only train the attention layers because they found it's more effective. So in the training process,  we only train the self-attn, the dropout is also for these layers.
